scripts/us_bls/jolts/BLSJolts.py

```diff
@@ -0,0 +1,197 @@
+
+jolts_df['Date'] = jolts_df.apply(period_year_to_iso_8601, axis=1)
+
+# Add relevant columns from series information
+jolts_df = jolts_df.merge(series_desc[['industry_code', 'region_code', 'seasonal', 'ratelevel_code']], left_on=["series_id"], right_index=True)
+
+# Drop rate data
+jolts_df = jolts_df.drop(jolts_df.query("ratelevel_code != 'L'").index)
+
+# Drop non-national data
+jolts_df = jolts_df.drop(jolts_df.query("region_code != '00'").index)
+
+# Map industry and seasonal adjustment to statistical variable name
+UNIQUE_INDUSTRIES = list(jolts_df['industry_code'].unique())
+ADJUSTED_TYPES = [("Adjusted", "BLSSeasonallyAdjusted"), ("Unadjusted", "BLSSeasonallyUnadjusted")]
+
+def mapRowToStatisticalVariable(row):
+  base_stat_var = row['statistical_variable']
+  industry_code = row['industry_code']
+  seasonal_adjustment = "Adjusted" if row['seasonal'] == "S" else "Unadjusted"
+
+  return f"dcs:{base_stat_var}_NAICS{industry_code}_{seasonal_adjustment}"
+
+# Build map to StatisticalVariable
+jolts_df['StatisticalVariable'] = jolts_df.apply(mapRowToStatisticalVariable, axis=1)
+jolts_df['Value'] = jolts_df['value']
+
+# Output final cleaned CSV
+FINAL_COLUMNS = ['Date', 'StatisticalVariable', 'Value']
+output_csv = jolts_df.loc[:, FINAL_COLUMNS]
+output_csv.to_csv("BLSJolts.csv", index=False, encoding="utf-8")
+
+### Step 3) Create StatisticalVariables  (6 job variables * 28 industries * 2 adjustments)
+SAMPLE_STAT_VAR = """
+Node: dcid:{STAT_CLASS}_NAICS{INDUSTRY}_{ADJUSTMENT}
+typeOf: StatisticalVariable
+populationType: {POPULATION}
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+measurementMethod: {BLS_ADJUSTMENT}
+naics: dcid:NAICS/{INDUSTRY}
+turnoverType: dcs:{TURNOVER_TYPE}
+"""
+
+# Output the schema mapping to a new file
+with open("BLSJolts_StatisticalVariables.mcf", "w", newline="") as f_out:
+  for schema_name, pop_type, sep_type, _ in schema_mapping:
+    for industry_code in UNIQUE_INDUSTRIES:
+      for adjusted_dcid_map, adjusted_schema in ADJUSTED_TYPES:
+        # Create new schema object
+        stat_var_schema = SAMPLE_STAT_VAR
+
+        # Remove separation type entry if not includes
+        if sep_type == "":
+          stat_var_schema = stat_var_schema.replace("turnoverType: dcs:{TURNOVER_TYPE}", "")
+
+        # Replace all other fields
+        stat_var_schema = stat_var_schema.replace("{STAT_CLASS}", schema_name)   \
+                                         .replace("{INDUSTRY}", industry_code)   \
+                                         .replace("{ADJUSTMENT}", adjusted_dcid_map)   \
+                                         .replace("{BLS_ADJUSTMENT}", adjusted_schema)   \
+                                         .replace("{POPULATION}", pop_type)      \
+                                         .replace("{TURNOVER_TYPE}", sep_type)
+
+        f_out.write(stat_var_schema)
+
+        if OUTPUT_MCF_TO_CONSOLE:
+          print(stat_var_schema)
+
+### Step 4) Create Template MCF
+BASE_MCFS = """
```

Comment:
They were originally dynamically created. I left them there in case we needed to add dynamic content in the future. I will move them to their own file.